CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Initialisation - Make Liana.init return a promise resolving when all is up.
```

Comment:
You could remove this since changelog will contain the commit text (with reference to this PR)